.changeset/odd-fans-wonder.md

```diff
@@ -0,0 +1,7 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer-backend': patch
```

Comment:
Before merging: This is a new feature, so please make this a `minor` change.

Reply:
Changed